docs/content/en/contributing/how-to-write-a-parser.md

```diff
@@ -36,7 +36,7 @@ $ docker-compose build --build-arg uid=1000
 |-------                                        |--------
 |`dojo/tools/<parser_dir>/__init__.py`          | Empty file for class initialization
 |`dojo/tools/<parser_dir>/parser.py`            | The meat. This is where you write your actual parser. The class name must be the Python module name without underscores plus `Parser`. **Example:** When the name of the Python module is `dependency_check`, the class name shall be `DependencyCheckParser`
-|`dojo/unittests/scans/<parser_dir>/{many_vulns,no_vuln,one_vuln}.json` | Sample files containing meaningful data for unit tests. The minimal set.
+|`unittests/scans/<parser_dir>/{many_vulns,no_vuln,one_vuln}.json` | Sample files containing meaningful data for unit tests. The minimal set.
```

Comment:
Hi @kiblik, good find. But I have seen there is one line missing for the actual unit test for a parser. Could you add that please, otherwise we would need another PR for this.

Reply:
Thank you